.circleci/continue_config.yml

```diff
@@ -264,8 +264,37 @@ jobs:
                 file: coverage.xml
             - store_test_results:
                 path: test-results
-
-  example-test:
+  example-test-dataset:
```

Comment:
Plz double check that the tests from this [PR](https://github.com/kolenaIO/kolena/pull/377) run as well